src/mqueryfront/src/QueryField.js

```diff
@@ -91,6 +94,37 @@ class QueryField extends Component {
     }
 
     render() {
+        var options = [];
+        var i;
+        for (i = 0; i < this.props.availableTaints.length; i++) {
+            options.push({
+                label: this.props.availableTaints[i],
+                value: this.props.availableTaints[i],
+            });
+        }
+
+        var multiselect = "";
```

Comment:
Avoid creating an empty span unnecessarily:
```suggestion
        var multiselect = null;
```